components/match2/wikis/ageofempires/match_summary.lua

```diff
@@ -69,7 +69,7 @@ function CustomMatchSummary.createBody(match)
 	end
 
 	Array.forEach(match.games, function(game)
-		if not game.map and not game.winner and String.isNotEmpty(game.resulttype) then return end
+		if not game.map and not game.winner and String.isNotEmpty(game.resultType) then return end
```

Comment:
looks like i failed more than once:
The goal was to display games with no map and winner=skip, thus resultType is np
```suggestion
		if not game.map and not game.winner and String.isEmpty(game.resultType) then return end
```